src/sentry/api/serializers/models/eventattachment.py

```diff
@@ -11,14 +13,24 @@ def get_attrs(self, item_list, user, **kwargs):
 
     def serialize(self, obj, attrs, user):
         file = attrs["file"]
+
         return {
             "id": str(obj.id),
+            "event_id": obj.event_id,
+            "type": obj.type,
             "name": obj.name,
-            "headers": file.headers,
-            "mimetype": obj.mimetype,
+            "mimetype": get_mimetype(file),
+            "dateCreated": obj.date_added,
```

Comment:
Does `obj.date_added` contain the same value as `file.timestamp`?

Reply:
yes it does, both use the current timestamp by way of defaults, as neither timestamp is explicitly set.